main/src/ca/uwaterloo/flix/language/ast/ParsedAst.scala

```diff
@@ -1223,6 +1223,16 @@ object ParsedAst {
       */
     case class Debug(sp1: SourcePosition, kind: ParsedAst.DebugKind, exp: ParsedAst.Expression, sp2: SourcePosition) extends ParsedAst.Expression
 
+    /**
+      * Instanceof expression (currently a binary op - maybe change the syntax?).
```

Comment:
update comment.

Reply:
Cut down to just `InstanceOf expression.`